src/test/java/com/uid2/client/PublisherTests.java

```diff
@@ -93,7 +93,8 @@ public void integrationBadRequests() {
         IdentityTokens currentIdentity = IdentityTokens.fromJsonString(PublisherTests.expectedDecryptedJsonForTokenGenerateResponse);
         Uid2Exception exception = assertThrows(Uid2Exception.class, //expired token
                 () -> publisherUid2Client.refreshToken(currentIdentity));
-        assertTrue(exception.getMessage().contains("expired"));
+        //We no longer assert on what the exception contains - since an expired key is later deleted, and deleted keys give a different error
+        //Previous code: assertTrue(exception.getMessage().contains("expired"));
 
```

Comment:
how come running the unit tests in PublisherTests.java give me NullPointerException?
![image](https://github.com/IABTechLab/uid2-client-java/assets/2499852/122a85ae-c65d-4349-bc8b-5ce4d85709be)

Reply:
integration tests have this requirement:
//this test requires these env vars to be configured: UID2_BASE_URL, UID2_API_KEY, UID2_SECRET_KEY